src/routes/_error.svelte

```diff
@@ -1,20 +1,22 @@
 <script>
-  import NotFound from "../components/NotFound.svelte";
+  import { onMount } from "svelte"; // eslint-disable-line import/no-extraneous-dependencies
+  import { goto } from "@sapper/app"; // eslint-disable-line import/no-extraneous-dependencies
 
   export let status;
   export let error;
 
+  onMount(() => {
```

Comment:
We don't need this, user can stay in the original path.

Reply:
Ok, I'll just revert to what it was before and display the NotFound component if the error is a 404

Reply:
Also, please add a descriptive page title like 'Page Not Found'.